experimental/dds/tree2/src/feature-libraries/editable-tree/editableTreeTypes.ts

```diff
@@ -325,6 +326,16 @@ export interface EditableField
 	 */
 	insertNodes(index: number, newContent: ITreeCursor | ITreeCursor[]): void;
 
+	/**
+	 * Moves nodes from a source to destination.
+	 */
+	moveNodes(
+		sourceIndex: number,
+		count: number,
+		destFieldPath: FieldUpPath,
+		destIndex: number,
```

Comment:
Use full words in names, especially in APIs. Also you don't need to repeat the types in the names:
```suggestion
		destinationField: FieldUpPath,
		destinationIndex: number,
```

Also since EditableField can't implement the FieldUpPath interface, using FieldUpPath is a bit annoying at this API level. I think taking in an EditableField for the destination will work better (and also make my typed API work easier).

Reply:
Also might be nice to make destinationField the last argument, so it can be optional, and have this default to a move within the current field if not provided.

Reply:
updated argument names, use `EditableField` type instead of `FieldUpPath`